store/storetest/post_store.go

```diff
@@ -1155,7 +1155,7 @@ func testPostStoreSearch(t *testing.T, ss store.Store) {
 	m1.ChannelId = c1.Id
 	m1.UserId = userId
 	m1.NotifyProps = model.GetDefaultChannelNotifyProps()
-	store.Must(ss.Channel().SaveMember(&m1))
+	if _, err := ss.Channel().SaveMember(&m1); err != nil {		panic(err)	}
```

Comment:
Same here, you should use something like:

```go
_, err := ss.Channel().SaveMember(&m1)
require.Nil(t, err)
```

And the same for any other case where you are replacing `store.Must`